conbench/entities/run.py

```diff
@@ -112,8 +112,8 @@ def get_baseline_run(
         self,
         commit_limit: int = 20,
         on_default_branch: bool = False,
-        case_id: str = None,
-        context_id: str = None,
+        case_id: Optional[str] = None,
+        context_id: Optional[str] = None,
```

Comment:
Oh wow, does `mypy` complain if you do this? I always thought `x: str = None` implied that `x` was actually `Optional[str]`. Maybe that's just how [sphinx-autodoc-typehints](https://github.com/tox-dev/sphinx-autodoc-typehints) works.

Reply:
:)